go/grpc/protocol-server.go

```diff
@@ -19,7 +19,7 @@ type ProtocolServer interface {
 
 func StartProtocolServer(server ProtocolServer, mdnsServer *mdns.Server, mdnsName string, port uint16) (uint16, error) {
 	host := fmt.Sprintf(":%d", port)
-	listener, err := net.Listen("tcp4", host)
+	listener, err := net.Listen("tcp", host)
```

Comment:
This enables ipv6 again, which made problems in the past. Is there a reason behind that change?